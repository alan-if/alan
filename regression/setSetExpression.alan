The l1 Isa location
  Has locationSet {l1}.
End The l1.

The l2 Isa location
End The l2.

Start At l1.
  Set locationSet Of l1 to {l1, l2}.
  "locationSet should = 2, actual =" Say Count In locationSet Of l1. ".$p"
  For Each loc In locationSet Of l1 Do
    Say loc.
  End For Each.
  "Should be ""l1 & l2"".$p"
