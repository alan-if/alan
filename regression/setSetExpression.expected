########## setSetExpression ##########

        No warnings or errors detected.


LocationSet should = 2, actual = 2.

L1 l2 Should be "l1 & l2".



L1

> 

Do you want to UNDO, RESTART, RESTORE or QUIT ? 
